util/topgen/validate.py

```diff
@@ -753,19 +753,19 @@ def check_power_domains(top):
         return error
 
     # check that power domain definition is consistent with reset and module definition
-    for reset in top['resets']['nodes']:
-        if reset['gen']:
-            if 'domains' not in reset:
-                log.error("{} missing domain definition".format(reset['name']))
-                error += 1
-                return error
-            else:
-                for domain in reset['domains']:
-                    if domain not in top['power']['domains']:
-                        log.error("{} defined invalid domain {}".format(
-                            reset['name'], domain))
-                        error += 1
-                        return error
+    #for reset in top['resets']['nodes']:
```

Comment:
will remove later when i'm sure this is working and the domain is also setting correctly.